x-pack/legacy/plugins/infra/public/pages/metrics/components/chart_section_vis.tsx

```diff
@@ -73,7 +73,9 @@ export const ChartSectionVis = ({
     ),
   };
 
-  if (!metric) {
+  if (!id) {
+    return null;
+  } else if (!metric) {
```

Comment:
@simianhacker @phillipb This was previously not reachable because the earlier condition caused an early return. Please let me know if I misinterpreted the intention of these conditions.

Reply:
🤦‍♂looks right to me